src/main/java/emu/grasscutter/game/shop/ShopManager.java

```diff
@@ -19,6 +19,8 @@
 public class ShopManager {
 	private final GameServer server;
 
+	public static Boolean DEBUG_ULTIMATED_SHOP = Boolean.FALSE;
```

Comment:
What is the purpose of this? You can always just give yourself enough resources to buy items.